docker/Dockerfile

```diff
@@ -32,7 +32,7 @@ FROM source${build} AS deps
 
 RUN yarn global add lerna && \
     lerna bootstrap -- --production --no-optional --ignore-scripts && \
-    rm -rf node_modules/@api3 && \
+    find -L node_modules/@api3 -maxdepth 1 -type d -name "airnode-*" | xargs rm && \
```

Comment:
@amarthadan I had to change this because @api3/promise-utils were removed.

I am glad that we have an e2e test running the client image in the pipeline :)